src/server/pegasus_event_listener.cpp

```diff
@@ -61,10 +62,13 @@ void pegasus_event_listener::OnCompactionCompleted(rocksdb::DB *db,
 
 void pegasus_event_listener::OnStallConditionsChanged(const rocksdb::WriteStallInfo &info)
 {
-    if (info.condition.cur == rocksdb::WriteStallCondition::kDelayed)
+    if (info.condition.cur == rocksdb::WriteStallCondition::kDelayed) {
+        derror_replica("rocksdb write delayed");
```

Comment:
Printing log every stalling, will it produce too many error logs?

Reply:
Yes. But this is a serious error. We print error logs on each serious error like 2PC write failure or read failure. Mostly write-stalls are followed by many write failures.